update_gmaster.sh

```diff
@@ -10,6 +10,7 @@ VFILE=$BASE/$PROJ/GVERSION
 date > $VTEMP
 UPSTREAM=`git rev-parse --abbrev-ref gupdater@{upstream}`
 REPO=${UPSTREAM%/*}
+BRANCHES=gupdater
```

Comment:
explicitly include gupdater below, where it also adds in master -- this will conflict with the config file that has BRANCHES defined.

Reply:
Done